### PR TITLE
Remove bloat

### PR DESCRIPTION
**Pull request by U2C9727A4**
I removed unused stuff and some other things i dont remember

### CODE REVIEW

File: packages.x86_64 [hunk 1]
```diff
@@ -122,15 +119,13 @@ xfsprogs
...
 
-# virbos
-#maxfetch
 # bspwm
 sway
 dmenu
 alacritty
 firefox
 xorg-xwayland
-xorg-xinit
```
Comment (jtbx) [inline packages.x86_64:133]: Why are you removing xinit?

File: packages.x86_64 [hunk 2]
```diff
@@ -122,15 +119,13 @@ xfsprogs
 xl2tpd
 zsh
 
-# virbos
-#maxfetch
```
Comment (jtbx) [inline packages.x86_64:126]: Could you please leave the maxfetch comment in, because that package is from our custom repo that doesn't work yet (https://github.com/Virbos/pkgs)

File: airootfs/etc/passwd [hunk 3]
```diff
@@ -1,2 +1 @@
 root:x:0:0:root:/root:/usr/bin/zsh
-liveuser:x:1000:1000::/home/liveuser:/usr/bin/zsh
```
Comment (jtbx) [inline airootfs/etc/passwd:2]: Why is the liveuser removed? That's the user that starts Xorg

File: airootfs/home/liveuser/.zlogin
Comment (jtbx) [inline airootfs/home/liveuser/.zlogin:1]: Why are these files removed, .zlogin runs startx, which runs .xinitrc, and i3 runs the i3 config. Also, the configs for i3 (and everything else) are stored in [virbos-configs](https://github.com/virbos/virbos-configs). The ISO build system pulls that repo and puts the configs where they need to be in the airootfs.

**Review by jtbx — CHANGES_REQUESTED** [review]

File: packages.x86_64 (hunk 1 above)
Comment (U2C9727A4) [reply packages.x86_64:133]: Since the ISO now runs sway with wayland it is no longer required

File: packages.x86_64 (hunk 2 above)
Comment (U2C9727A4) [reply packages.x86_64:126]: Alright, ill be editing it to add a commented maxfetch

File: airootfs/etc/passwd (hunk 3 above)
Comment (U2C9727A4) [reply airootfs/etc/passwd:2]: Liveuser is unable to start sway with wayland due to lack of permissions so it was removed

File: airootfs/home/liveuser/.zlogin
Comment (U2C9727A4) [reply airootfs/home/liveuser/.zlogin:1]: These files were removed because liveuser was removed. Sway is started from the .zlogin of root

**Comment by jtbx** [comment]
I think we'll keep liveuser but add a check to set the environment variable WLR_NO_HARDWARE_CURSORS if the ISO is running in KVM. This should make the ISO in a VM work a little better, and also I'll try fix any permission errors under liveuser.
